R/check_suggested.R

```diff
@@ -39,22 +37,34 @@ check_suggested <- function(packages,
     return(FALSE)
   }
 
-  # If TRUE, fail if package is not installed
-  for (pkg in packages) {
-    if (!is_installed(pkg)) {
-      if (is_testing()) {
-        if (is.null(message)) {
-          message <- glue("`{top_level_fun}()` needs the '{pkg}' package.")
-        }
-        testthat::skip(message)
-      } else {
-        if (is.null(message)) {
-          message <- glue("to use `{top_level_fun}()`.")
-        }
-        # Could use the version argument too.
-        rlang::check_installed(packages, message)
-      }
+  # Return early if all packages are installed.
+  if (rlang::is_installed(packages)) {
+    return(TRUE)
+  }
+
```

Comment:
```suggestion
```

Not needed if we do an early return?

Reply:
I tried playing with the order without chaning test results, but it didn't work.

Reply:
Possibly in a follow-up because this PR aims not to change the default behavior. (except gor removing the version and message argument from check_suggested